File: RMS(Server)/RMS(Server)/Models/CashFlow.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace RMS_Server_.Models
{
    public class CashFlow
    {
        public int Id { get; set; }
        public Order Order { get; set; }
        public string OrderId { get; set; }
        public int OrderPrice { get; set; }
        public double InventoryCost { get; set; }
        public double Profit { get; set; }
        public Inventory Inventory { get; set; }
        public string InventoryId { get; set; }
    }
}

File: RestaurantManagementApp/src/app/shared/cash-flow.ts
import {Order} from './order.model';
import {Inventory} from './inventory.model';

export class CashFlow {
  public Id: number;
  public Order: Order[];
  public OrderId: string;
  public OrderPrice: number;
  public InventoryCost: number;
  public Profit: number;
  public Inventory: Inventory[]  ;
  public InventoryId: string;

  constructor(id: number, order: Order[], orderId: string, orderPrice: number,
              inventoryCost: number, profit: number,
              inventory: Inventory[], inventoryId: string) {

    this.Id = id;
    this.Order = order;
    this.OrderId = orderId;
    this.OrderPrice = orderPrice;
    this.InventoryCost = inventoryCost;
    this.Profit = profit;
    this.Inventory = inventory;
    this.InventoryId = inventoryId;
  }
}
